tools/java/org/unicode/cldr/test/CheckForExemplars.java

```diff
@@ -497,6 +497,8 @@ private String checkAndReplacePlaceholders(String path, String value, List<Check
             }
         }
 
+        // TODO: move these tests to CheckPlaceholder
```

Comment:
Noticed some duplication that could be cleaned up, but didn't want to muddy the waters now.